trpcultivate_phenotypes/src/TripalCultivateValidator/ValidatorTraits/ValidValues.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Drupal\trpcultivate_phenotypes\TripalCultivateValidator\ValidatorTraits;
+
+/**
+ * Provides setters focused on configuring a validator that processes the columns
+ * in a file row and checks the value in the column against a list of valid values
+ */
+trait ValidValues {
+
+  /**
+   * Sets the array of values for which the validator is checking for in a
+   * specified set of columns (set by another validator trait) within a file row
```

Comment:
What about this wording:


```suggestion
   * Sets a list of values that should be allowed for a cell being validated.
   *
   * Note: Often this setter is combined with the setIndices() setter which indicates
   * which columns should have one of these values.
```